wazuhapp/Docker/build.sh

```diff
@@ -43,22 +43,22 @@ prepare_env() {
 
 download_sources() {
 
-    if ! curl -L https://github.com/wazuh/wazuh-kibana-app/tarball/${wazuh_branch} | tar zx ; then
-        echo "Error downloading the source code from GitHub."
-        exit 1
-    fi
+
 
     if ! curl -L https://github.com/elastic/kibana/tarball/v${kibana_version} | tar zx ; then
         echo "Error downloading Kibana source code from GitHub."
         exit 1
     fi
 
     mv elastic-* kibana_source
-    mkdir -p kibana_source/plugins/wazuh
-    mv wazuh-* wazuh_source
-    mv wazuh_source/* kibana_source/plugins/wazuh
-    mv kibana_source ${kibana_dir}
+    mkdir -p kibana_source/plugins
 
+    if ! git clone https://github.com/wazuh/wazuh-kibana-app.git --branch ${wazuh_branch} --single-branch kibana_source/plugins/wazuh ; then
```

Comment:
```suggestion
    if ! git clone https://github.com/wazuh/wazuh-kibana-app.git --branch ${wazuh_branch} --depth=1 kibana_source/plugins/wazuh ; then
```
This will clone only the last comment of `${wazuh_branch}`, making this faster than cloning the full branch.

Reply:
I updated the code